packages/components/src/Switch/Switch.tsx

```diff
@@ -163,28 +165,24 @@ const PlainSwitch = ({
 };
 
 export const Switch: React.FC<SwitchProps> = ({
-  checked,
-  onChange,
   label,
   disabled,
   variant: variantProp,
-  name,
-  value,
-  ...marginProps
+  width: widthProp,
```

Comment:
could also be only `w`, right? `WidthProps` should bring multiple props

Reply:
Nope, the WidthProps only accept "width", not "w".